modules/results/src/main/twirl/views/resultpanels/hhpred/forward.scala.html

```diff
@@ -1,13 +1,24 @@
 @import de.proteinevolution.models.results.FileLoader
 
-@(filepath : String)()
+@(filepath : String, jobID: String)()
```

Comment:
please rename the variable to `parentId` so that it is clear what purpose the variable has.

Reply:
It is not yet a parentId at that point. It will become a parentId after forwarding.